src/componenets/App/App.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { connect } from 'react-redux';
+import './App.scss';
+import { fetchMovies } from '../../actions';
+
+class App extends Component {
+  constructor() {
+    super();
+    // this.state
+  }
+
+  componentDidMount() {
+    this.props.dispatch(fetchMovies())
+  }
+
+  render() {
+    const { error, loading, movies } = this.props;
+
+    if (error) {
+      console.log(error);
+    }
+
+    if (loading) {
+      console.log(loading);
+    }
+    
+    // this.props.movies.movies.map(movie => {
```

Comment:
I think we're really close on this piece that was hanging us up. I will mess with it tonight and we can ask for help tomorrow if we can't figure it out